modules/cluster-logging-deploy-clo.adoc

```diff
@@ -95,155 +36,30 @@ metadata:
-subscribe a Namespace to an Operator.
-+
-.Example Subscription
-[source,yaml]
-----
-apiVersion: operators.coreos.com/v1alpha1
-kind: Subscription
-metadata:
-  generateName: "elasticsearch-"
-  namespace: "openshift-operators-redhat" <1>
-spec:
-  channel: "4.3" <2>
-  installPlanApproval: "Automatic"
-  source: "redhat-operators"
-  sourceNamespace: "openshift-marketplace"
-  name: "elasticsearch-operator"
-----
-<1> You must specify the `openshift-operators-redhat` namespace.
-<2> Specify `4.3` as the channel.
-
-.. Create the Subscription object:
-+
-----
-$ oc create -f <file-name>.yaml
-----
-+
-For example:
-+
-----
-$ oc create -f eo-sub.yaml
-----
-
-.. Change to the `openshift-operators-redhat` project:
-+
-----
-$ oc project openshift-operators-redhat
-
-Now using project "openshift-operators-redhat"
-----
-
-.. Create a Role-based Access Control (RBAC) object file (for example, `eo-rbac.yaml`) to grant Prometheus permission to access the `openshift-operators-redhat` namespace:
-+
-[source,yaml]
-----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
-metadata:
-  name: prometheus-k8s
-  namespace: openshift-operators-redhat
-rules:
-- apiGroups:
-  - ""
-  resources:
-  - services
-  - endpoints
-  - pods
-  verbs:
-  - get
-  - list
-  - watch
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: RoleBinding
-metadata:
-  name: prometheus-k8s
-  namespace: openshift-operators-redhat
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: Role
-  name: prometheus-k8s
-subjects:
-- kind: ServiceAccount
-  name: prometheus-k8s
-namespace: openshift-operators-redhat
-----
-
-.. Create the RBAC object:
-+
-----
-$ oc create -f eo-rbac.yaml
-----
-+
-The Elasticsearch operator is installed to the `openshift-operators-redhat` namespace and copied to each project in the cluster.
-
-. Install the Cluster Logging Operator using the {product-title} web console for best results:
+. Install the Cluster Logging Operator:
 
 .. In the {product-title} web console, click *Operators* -> *OperatorHub*.
 
 .. Choose  *Cluster Logging* from the list of available Operators, and click *Install*.
 
-.. On the *Create Operator Subscription* page, under *A specific namespace on the cluster* select *openshift-logging*.
+.. On the *Create Operator Subscription* page, under *A specific Namespace on the cluster* select *openshift-logging*.
 Then, click *Subscribe*.
 
 . Verify the operator installations:
 
 .. Switch to the *Operators* → *Installed Operators* page.
 
 .. Ensure that *Cluster Logging* is listed in the *openshift-logging* project with a *Status* of *InstallSucceeded*.
-
-.. Ensure that *Elasticsearch Operator* is listed in the *openshift-operator-redhat* project with a *Status* of *InstallSucceeded*.
-The Elasticsearch Operator is copied to all other projects.
 +
 [NOTE]
 ====
 During installation an operator might display a *Failed* status. If the operator then installs with an *InstallSucceeded* message,
```

Comment:
Out of scope, but:
s/During installation an operator might/During installation, an Operator might/

Reply:
Thank you. I didn't realize we always capitalize `Operator`?  I was thinking here operator is generic, so no cap. Like `the president` vs the President of the United States`.  Fixed!